tempesta_fw/http_parser.c

```diff
@@ -864,64 +874,47 @@ __parse_transfer_encoding(TfwHttpMsg *msg, unsigned char *data, size_t *lenrval)
  *
  * tfw_cache_copy_resp() also must be updated.
  */
-static void
-__store_header(TfwHttpMsg *hm, unsigned char *data, long len, int id,
-	       bool close)
+static TfwHttpHdr *
+__alloc_header(TfwHttpMsg *hm, tfw_http_hdr_t id)
 {
 	TfwHttpHdrTbl *ht = hm->h_tbl;
-	TfwStr *h;
 
-	if (unlikely(id == TFW_HTTP_HDR_RAW
-		     && hm->h_tbl->off == hm->h_tbl->size))
+	if (unlikely(id == TFW_HTTP_HDR_RAW))
 	{
```

Comment:
Place braces at the same line with IF statement if the statement is single-line.